### PR TITLE
Cleanup stuff

### PR DESCRIPTION
**Pull request by sordonia**
I cleaned up a lot of unused json configs, scripts and wrote the README in such a way to be able to train Private, MBC and run Arrow. 

More work is surely needed here so please chip in!

### CODE REVIEW

File: mttl/models/expert_config.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def _set_defaults(self):
         self.eval_mmlu_few_shot = True  # use few-shot for mmlu, default
         self.eval_mmlu_flag = False  # eval mmlu performance during training
         self.eval_rouge_flag = False  # eval rouge during training
-        self.pipeline_eval_tasks = "all"
+        self.pipeline_eval_tasks = None
```
Comment (matheper) [inline mttl/models/expert_config.py:66]: Just double checking.. Is the idea of this change to have eval disabled by default, or to use all if `pipeline_eval_tasks = None`

Comment (shuishen112) [reply mttl/models/expert_config.py:66]: I think setting "all" as default is better. "None" causes "no task find" error.

Comment (sordonia) [reply mttl/models/expert_config.py:66]: is to don't eval on tasks by default, does this work?

**Review by pclucas14 — APPROVED** [review]
LGTM

**Review by matheper — APPROVED** [review]

**Pull request merged**